.travis.yml

```diff
@@ -9,15 +9,32 @@ python:
   - "3.5"
   - "pypy"
   - "pypy3"
+matrix:
+  include:
+    - python: "2.7"
+      env: TOXENV=perf
 
-install:
-  - pip install -e .
-script:
-  - py.test
+before_install: |
+  pip install tox "virtualenv<14.0.0" # recent versions don't run on Python 3.2
```

Comment:
Okay, sounds good. pytest adopted the policy "we can drop it as soon as pip does", so we could drop it, but I don't think the need came up so far. Honestly I'm not sure how exactly pytest handles the text/unicode stuff so it's compatible with 2.6-3.5 without any major pains.